activelearning.py

```diff
@@ -1,349 +1,76 @@
+"""
+Active learning with GFlowNets
+"""
+
 from omegaconf import DictConfig, OmegaConf
-import hydra
+from oracle import Oracle
+from proxy import Proxy
+from acquisition import AcquisitionFunction
+from env import Env
+from gflownet import GFlowNet
+from querier import Querier
+
 
-#--------------------------------
-'''
-AL Global pipeline
-'''
-#TODO : instantiate the config with hydra . Once we have the config object we can pass it to other functions
+# TODO : instantiate the config with hydra . Once we have the config object we can pass it to other functions
 class ActiveLearning:
+    """
+    AL Global pipeline : so far it is a class (the config is easily passed on to all methods), very simple, but it can be a single simple function
+    """
```

Comment:
I do not think it is fully justified to make this a class, but let's leave it for now. I would just have `main()` function that runs the pipeline given a configuration.